src/models/StationModel.tsx

```diff
@@ -0,0 +1,8 @@
+export interface StationModel {
+    name: string;
+    crs: string;
+}
+
+export const areStationsEqual = ( station1: StationModel, station2: StationModel ) => {
```

Comment:
within models we don't want anything that isn't a model, I'd pop this in a stations helper file

Reply:
also no need for spacing around normal brackets